ledfx/config.py

```diff
@@ -77,11 +77,12 @@ def wled_optional_generator(setting):
         vol.Optional("user_presets", default={}): dict,
         vol.Optional("scenes", default={}): dict,
         vol.Optional("integrations", default=[]): list,
+        vol.Optional("transmission_mode", default=0): int,
```

Comment:
I'd prefer this to be a descriptor so that we can identify in the future what it means.

Also would allow us to have additional transmission_modes in the future (for example, gzip+b64)

If we leave some breadcrumbs here what 0 and 1 mean that is also OK - just want to document what we're doing

Reply:
Descriptor as in change it to dict or just add a description?

Reply:
A dict is probably fine (we don't care about payload size for the config) - "uncompressed" and "compressed" maybe?

Reply:
Perhaps a string is good enough? Because I think dict may be a bit redundant, since then the config would be called transmission_mode and the dict would be something like `mode: uncompressed`.

Reply:
Yeah just a string is fine - you can define the options as a list.

Reply:
also i can name it in the frontend however you like.
(value does not need to be === label)

Reply:
Changed it so that the config would take "compressed" and "uncompressed". Also, rather than keep it in a list, added a class similar to how it is done in events.py, makes it look a bit more clear I think. If that's not alright, let me know and I will change it.

Reply:
Oh also forgot to mention, made compressed case default. Also should we consider going with pybase64 or is native base64 good enough?